pages/components/docs/Flex.md

```diff
@@ -1,41 +1,36 @@
 # Flex and Flex.Item
 
-Flex and Flex.Item are wrapping components that will give the content flexbox properties. Flex.Item is included in the export for Flex.
+Flex is a component that will allow you to use flex box properties
+
+*** Previously, `Flex.Item` was used for flex item specific properties - we've added all properties to the `Flex` component, but are keeping Flex.Item around for backwards compatibility. ***
```

Comment:
Does `***` mean anything to the markdown parser?

Reply:
Yeah I changed it to just one star *for italic text*